src/ui/icons/SvgIcon.stories.tsx

```diff
@@ -0,0 +1,47 @@
+import { ComponentStory, ComponentMeta } from '@storybook/react';
+
+import { Box, ArrowDownLeftIcon, Text } from '../index';
+
+export default {
+  title: 'Design System/Components/SvgIcon',
+  component: Box,
+} as ComponentMeta<typeof Box>;
+
+const Template: ComponentStory<typeof Box> = args => (
+  <Box
+    css={{
+      display: 'flex',
+      flexDirection: 'column',
+      gap: '$sm',
+    }}
+  >
+    {args.children}
+  </Box>
+);
+
+export const SingleArrowDownLeftIcon = Template.bind({});
+
+SingleArrowDownLeftIcon.args = {
+  children: (
+    <>
+      <Box css={{ display: 'flex', gap: '$sm' }}>
+        <Text>{`xs (10px)`}</Text>
```

Comment:
```suggestion
        <Text>xs (10px)</Text>
```
same with the other 3